start-from-zero/index.css

```diff
@@ -1,17 +1,47 @@
+@import 'bootstrap-grid-only.css';
```

Comment:
Hmm, this is very interesting. I refactored again my sass files with the 7-1 architecture pattern -> https://www.sitepoint.com/architecture-sass-project/  I hope this will fix the problem.